File: ed1/finale/nonna/att/nonna.c
#include <stdio.h>
#include <assert.h>

#define MAXN 5000
#define MAXK 5000
#define MAXP 1000000

int mangia(int N, int K, int P[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


int P[MAXN];

int main() {
    FILE *fr, *fw;
    int N, K, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(2 == fscanf(fr, "%d %d", &N, &K));
    for(i=0; i<N; i++)
        assert(1 == fscanf(fr, "%d", &P[i]));

    fprintf(fw, "%d\n", mangia(N, K, P));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: ed1/finale/nonna/att/nonna.pas
function mangia(N, K: longint; var P: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    mangia := 42;
end;


const
    MAXN = 5000;
    MAXK = 5000;
    MAXP = 1000000;
var
    N, K, i : longint;
    P       : array[0..MAXN-1] of longint;
    fr, fw  : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N, K);
    for i:=0 to N-1 do
        read(fr, P[i]);

    writeln(fw, mangia(N, K, P));
    close(fr);
    close(fw);
end.
